Sources/Basics/Exports.swift

```diff
@@ -0,0 +1,14 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2021 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import OrderedCollections
+
+public typealias OrderedDictionary = OrderedCollections.OrderedDictionary
+public typealias OrderedSet = OrderedCollections.OrderedSet
```

Comment:
What's the benefit of this file (no `@_exported` for the import compared to #3590) over manually importing `OrderedCollections`?

Reply:
good point. @neonichu @abertelrud do you think re-exporting common modules like swift-collections, swift-system would make sense, or prefer individual imports? I am on the fence.

Reply:
Personally not a fan of re-exporting, it might also hurt incremental builds?

Reply:
I strongly prefer individual imports, to make it clear what's being used.  Tooling should ideally help add those where needed (the effort of having to add the individual imports is usually the main argument against it).  So I agree with neonichu on this.

Reply:
It also makes it clear that the standard ordered collections are being used rather than something specific to TSC.

Reply:
Unless I'm reading it wrong, it seems like the general opinion is not to import `OrderedCollections` via `Basics`, but to import it individually in files that need it. If this is the case, @tomerd could you remove this `Export` file and import `OrderedCollections` individually before this PR is merged?